client/src/components/ServiceHoursCard.tsx

```diff
@@ -7,7 +7,7 @@ import vector from '../images/vector.png';
 import { BASE_URL } from '../constants';
 
 export default function ServiceHoursCard({ parentToChild }: {parentToChild: string}): ReactElement {
-	const id = parentToChild.replace(/\W+/g, '-').replace(/-$/, '').toLowerCase();
+	const id = parentToChild;
```

Comment:
Maybe replace the `parentToChild` variable with something that has more meaning, like `id` or `serviceId`